### DISCUSSION TITLE
Connection Issues

### DISCUSSION THREAD

**Discussion by spudgun1001** [hnimminh/libresbc#168] [Q&A]
I am using the docker version of Libresbc and I have these issues popping up when running the container.

![image](https://github.com/user-attachments/assets/072e58da-9aaf-475e-a278-f570cc1436f1)

I have managed to get the webui to work, however I get the error that i cannot connect to port 8443.

Any ideas on why this is happening and how to fix?

**Comment by git-root** [comment]
@spudgun1001 I was facing the same issue. the issue is in docker platform,  redis is expected to run inside the docker itself. there is a check 
https://github.com/hnimminh/libresbc/blob/9438a58b72626caa7f9c8a06b931494694b86a28/liberator/basemgr.py#L391 
if not LIBRE_REDIS:
        logger.info(f"module=liberator, space=basemgr, action=rdbinstance, message=[skip action since buitin redis is disabled]")
        return
So i changed in libre.env as below to make it work        
--- a/build/docker/libre.env
+++ b/build/docker/libre.env
@@ -5,4 +5,4 @@ SCAN_COUNT=1000
 REDIS_TIMEOUT=5
 NODEID=devsbc
 LOGSTACKS=CONSOLE
-LIBRE_REDIS=NO
+LIBRE_REDIS=YES
